services/workflows-service/src/env.ts

```diff
@@ -1,3 +1,32 @@
-// Identifiers to pass into NestJS's ConfigService
+import 'dotenv/config';
+import { createEnv } from '@t3-oss/env-core';
+import { z } from 'zod';
 
-export const JWT_EXPIRATION = 'JWT_EXPIRATION';
+export const env = createEnv({
+  /*
+   * clientPrefix is required.
+   */
+  clientPrefix: 'PUBLIC_',
+  server: {
+    NODE_ENV: z.enum(['development', 'production', 'test']),
+    ENV_FILE_NAME: z.string().optional(),
+    BCRYPT_SALT: z.coerce.number().int().nonnegative(),
```

Comment:
This validation dosent match the original, string is acceptable

Reply:
The salt tests are checking that it is a number and is above 0. There was also parseInt in use prior to my changes (zod now converts to number)